api/v1beta2/awscluster_types.go

```diff
@@ -208,10 +208,9 @@ type AWSLoadBalancerSpec struct {
 	// +optional
 	AdditionalSecurityGroups []string `json:"additionalSecurityGroups,omitempty"`
 
-	// AdditionalIngressRules sets the additional ingress rules for the control plane load balancer. If no source security group ids are specified, the
-	// default control plane security group will be used.
+	// IngressRules sets the ingress rules for the control plane load balancer.
```

Comment:
Do we still want to indicate that if not provided the defaults will be used?

Reply:
> If no source security group ids are specified, the default control plane security group will be used.

I don't think so because that is not true anymore.

Reply:
Isn't this the default though? https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/4304/files#diff-59c59dbf183fdde59cf0030bd23388381951860f0002a073a55f330c4733f8ddR583-R602

Reply:
I may be miss understanding the original code comment so bear with me for a second :smile: 

The code comment says

> If no source security group ids are specified, the default control plane security group will be used.

But in the current implementation, there is no defaulting for the source security group of the ingress rules. The ingress rules are used "as is", without defaulting the source of the traffic. That's why I believe the comment is not true anymore.